specification/devcenter/resource-manager/readme.md

```diff
@@ -87,7 +98,6 @@ swagger-to-sdk:
   - repo: azure-sdk-for-java
   - repo: azure-sdk-for-js
   - repo: azure-cli-extensions
-  - repo: azure-sdk-for-go
```

Comment:
why we remove this?
If no particular reason, we should revert this change

Reply:
I did not see this change on the linked private PR, therefore please revert this